sdk/storage/storage-blob/recordings/browsers/aborter/recording_should_abort_after_aborter_timeout.json

```diff
@@ -14,8 +14,9 @@
  ],
  "uniqueTestInfo": {
   "uniqueName": {
-   "container": "container157835297384300414"
+   "container": "container159439924380601426"
   },
   "newDate": {}
- }
+ },
+ "hash": "f8c932174e6354a067cc7325a6415091"
```

Comment:
@HarshaNalluru I noticed that now hash is added.  Is it because these recording was created before we introduced the hash? If I understand correctly the hash only affect soft-record mode.  Without it we'd always record but everything else should be the same.

Reply:
Yes, you're right. The hash is added for soft-record. This recording is old and doesn't have the hash.
To be compatible with the old recordings and not re-record everything just for the hash, soft-record falls back to record if the hash is outdated or absent.

Reply:
Reverted changes to recordings as they are not relevant to this PR.